v2/pkg/api/v1alpha2/types_config.go

```diff
@@ -65,6 +65,32 @@ type Platform struct {
 	Release string `json:"release,omitempty"`
 }
 
+func (p Platform) DeepCopy() Platform {
+	copy := Platform{
+		Graph: p.Graph,
+	}
+
+	copy.Channels = make([]ReleaseChannel, len(p.Channels))
+	for i, ch := range p.Channels {
```

Comment:
In case all elements are important to be copied, it is possible to use copy or append method instead of running the for and handling all fields.

If only few fields are required, please ignore the comment above.